public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -26,38 +28,99 @@ export interface GenAIButtonProps {
 export const GenAIButton = ({
   text = 'Auto-generate',
   loadingText = 'Generating',
+  toggleTipTitle = '',
   onClick: onClickProp,
   messages,
   onGenerate,
   temperature = 1,
 }: GenAIButtonProps) => {
```

Comment:
Should history feature be optional? i.e. do we want this experience in all cases? IMO for dashboard save as changes users may not want to workshop this as they might panel title / description

Reply:
Actually, I was thinking the other way around 😅 It could be useful to add refinements like: "do not mention panel names", "group the changes by panel", "discard X changes", etc